source/CardGames.py

```diff
@@ -250,6 +250,9 @@ def rewardPot(self, player):
     player.addMoney(self.money)
     self.money = 0
 
+  def resetPot(self):
```

Comment:
doesn't the rewardPot function already reset the pot?

Reply:
Yes, but there are situations where we may want to reset the pot without actually rewarding the money to anyone, such as if the game was ended prematurely.